Source/UI/NewUI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -55,13 +55,18 @@ final class BillingFormViewController: UIViewController {
         view.allowsSelection = false
         view.translatesAutoresizingMaskIntoConstraints = false
         view.register(CellTextField.self)
+        view.register(CellButton.self)
+
         return view
     }()
 
     // MARK: - UIViewController
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        if #available(iOS 13.0, *) {
+            isModalInPresentation = true // available in IOS13
```

Comment:
I am not 100% sure on this lifecycle, but `viewDidLoad` might be called as part of the presentation cycle, so by the time you tell it to be modal, the presentation has already started and this won't be in effect. Any difference if you set this at use site (to avoid complicating with an initialiser - they are painful for UI controllers).

Reply:
please suggest code using <cmd+g>